js/blocks/controls/index.js

```diff
@@ -22,4 +23,5 @@ export default {
 	select: BlockLabSelectControl,
 	multiselect: BlockLabMultiselectControl,
 	toggle: BlockLabToggleControl,
-}
\ No newline at end of file
+	'color-picker': BlockLabColorPickerControl,
```

Comment:
Minor gripe here, but can we wrap all the keys in `'single quotes'` if we're going to wrap `'color-picker'`?

Reply:
Actually, a better, simpler solution would be to refer to it simply as `color`. e.g.:
`color: BlockLabColorControl`

This would require renaming the PHP class as well.